plugins/examples/headlamp-events/README.md

```diff
@@ -0,0 +1,18 @@
+# Example Plugin: Reacting to Headlamp events
```

Comment:
Some of these sentences are marked hard and very hard to read. 

See...
![image](https://github.com/headlamp-k8s/headlamp/assets/9541/0c369e44-17c1-4afb-ba6c-16485f07598f)

Reply:
I have used this app in the past and to be honest I find it very difficult to write sentences it likes while still keeping a speech that is technical and resembles my own "voice". So I prefer to get yours and other readers' own review/suggestions when sentences sound complex. Maybe this is my problem or maybe I am trying to reach the best state in it and that that's too demanding.

However, I do agree these texts you pointed out sounded weird. I did an effort and used the app for the examples you mentioned and hopefully it's better now (though not perfect, according to the app 🙂 ).